_posts/2020/2021-01-07-how-to-animate-a-svg-with-css.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: How to animate SVGs with CSS
+date: 2021-01-07
+header_image: public/animate-svg-css.jpg
+header_position: bottom
+header_overlay: true
+category: coding
```

Comment:
Did you check back with Svenja on that? Not sure if it really fits in the coding category...